files/docker/grest/scripts/postgres-entrypoint.sh

```diff
@@ -0,0 +1,385 @@
+#!/usr/bin/env bash
+
+###################### Customisations - START ##################################
+apt-get update  > /dev/null 2>&1
+apt-get install -y socat curl gawk jq sudo postgresql-13-pglogical > /dev/null 2>&1
+apt-get install -y --no-install-recommends cron  > /dev/null 2>&1
+curl https://access.2ndquadrant.com/api/repository/dl/default/release/deb | bash
+
+# To add a user without a password the command is the following:
+# adduser --disabled-password --gecos '' guild#
+sed -i 's/%sudo.*/%sudo   ALL=(ALL) NOPASSWD:ALL/g' /etc/sudoers > /dev/null 2>&1
+adduser postgres sudo  > /dev/null 2>&1
+
+
+# Sets the postgres user cronJobs
+chown postgres:root /etc/cron.d/crontab   > /dev/null 2>&1
+chmod 0660 /etc/cron.d/crontab   > /dev/null 2>&1
+touch /var/log/cron.log > /dev/null 2>&1
+chmod 0660 /var/log/cron.log > /dev/null 2>&1
+chown postgres:root /var/log/cron.log > /dev/null 2>&1
+crontab -u postgres /etc/cron.d/crontab  > /dev/null 2>&1
+sudo -u postgres crontab &
```

Comment:
If we setup the cron jobs directly from repo files, do we need sudo permissions for postgres user?

It would be nice to make the docker setup make the cron files available, but we have the option to activate them in the setup-grest stage (so we don't run cron jobs during syncing).
@redoracle

Reply:
We can place it wherever you like, as long as we automate everything.
Docker env are not really born to manually do stuff.
We do have our koios env to query what it is last epoch and by comparing it with the local epoch number we can trigger the action to run the query setup including the cronjobs.
Personally I like modular things, putting everything in setup-postgres.sh (which still have problems) is not my favourite choice.